arcgis-ios-sdk-samples/Maps/Change map view background/README.metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "category": "Maps",
-    "description": "Customize map view\u2019s background by changing its grid properties.",
+    "description": "Customize map view's background by changing its grid properties.",
```

Comment:
Please also replace the [\u2019](https://www.fileformat.info/info/unicode/char/2019/index.htm) character in the README 😏

Reply:
Not sure what you're referring to, the README doesn't have the \u2019 character.

Reply:
Haha I know this is tricky.

![image](https://user-images.githubusercontent.com/9660181/82716064-8f112580-9c4a-11ea-811a-44e7c57976a3.png)

In the readme that character is a unicode right single quote mark, rather than an ASCII single quote mark. It is hard to tell visually 😏